internal/keepers/factory.go

```diff
@@ -60,6 +60,8 @@ func (d *keepersReportingFactory) NewReportingPlugin(c types.ReportingPluginConf
 		return nil, types.ReportingPluginInfo{}, fmt.Errorf("%w: failed to decode off chain config", err)
 	}
 
+	d.logger.Printf("successfully decoded offchain config when creating plugin: %s", string(c.OffchainConfig))
```

Comment:
will a string conversion like this work directly? is there a better way to print a struct

Reply:
you can use %v and the struct directly

Reply:
The initial approach would just print a JSON string. I'm fine with both. Seems like you already pushed the update so can we merge it?